### ISSUE TITLE
Reverse zoom

### ISSUE THREAD

**Issue by wdvr** [heartexlabs/label-studio-frontend#135]
The image zoom direction by scrolling seems to be reverse of what other applications use:

label studio: 
- ctrl+ scroll up == zoom out
- ctrl + scroll down == zoom in

other software: 
- scroll up == zoom in
- scroll down == zoom out

Two suggested solutions:
1. change it for everyone, by changing the sign here: https://github.com/heartexlabs/label-studio-frontend/blob/fd1e7cad9fdab9ca3c55691b2d0ee23905d51d59/src/tools/Zoom.js#L121
2. make it configurable by the end user - with an xml parameter similar to zoomControl. See https://github.com/heartexlabs/label-studio-frontend/pull/134 for a first proposal for this.

**Comment by hlomzik** [comment]
Hi! Sorry for late response, we've been working on a huge rework.
Scrolling direction is always a big question since Apple introduced inverted scroll :) I think the better option would be to add the settings because this behaviour should be changed per labeller (according to their preferences and environment), not per project. Also it would be good to detect preferred direction automatically, I'll think about it.
And thank you for PR!

**Comment by MichalLebeda** [comment]
Any updates on this? I am also a proponent of a user setting in this case

**Comment by makseq** [comment]
@MichalLebeda Which LS version do you use?

**Comment by MichalLebeda** [comment]
@makseq I use v1.6.0. I use CTRL + touchpad pinch gesture to zoom on Windows 10. It zooms in the opposite direction to the pinch gesture (unlike Google Maps for example).
There is also a problem when not being in the image bounding box, pinch gesture zooms the whole page (browser handles the event probably). I specifically mean the case when the cursor is next to the image in an empty space which is created by zooming the image out. Image is always left aligned and it grows only to the right direction.

**Comment by hlomzik** [comment]
Yes, that's still the case, but we'll address it soon. There will be a parameter in settings and most probably the default value will be the opposite one (the correct one). One more thing that it can be OS dependant só we'll check it.
